html/pfappserver/lib/pfappserver/I18N/i_default.po

```diff
@@ -6996,44 +6996,44 @@ msgstr "pending"
 msgid "pf::pki_provider::packetfence_pki"
 msgstr "PacketFence PKI"
 
-# pf::filter (Portal Profile Filters)
-msgid "pf::profile::filter::connection_type"
+# pf::condition::profile (Portal Profile Filters)
+msgid "pf::condition::profile::connection_type"
```

Comment:
This is still used for localizing all the profile filter condition types.
So should move to either localize short name such as 'connection_type'
Or keep the prefix so we can have the ability to localize each profile filter condition on it's own.